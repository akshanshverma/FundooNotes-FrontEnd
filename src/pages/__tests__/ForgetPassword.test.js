import React from 'react';
import { shallow } from 'enzyme';

import ForgetPassword from "../ForgetPassword";


const component = shallow(<ForgetPassword debug />);

describe('forgetPassword',()=>{
    it('should render correctly in "debug" mode', () => {      
        expect(component).toMatchSnapshot();
    });
    // it('renders a email input', () => {
    //     expect(component.find('#emailForPasswordReset').length).toEqual(1);
    // });

    it('should change event and email set in state to send reset password link', () => {
        const emailInput = component.find('#emailForPasswordReset');
        emailInput.props().onChange({
            target: {
                name: 'email',
                value: 'akshansh.verma01@gmail.com'
            }
        });
        expect(component.state('email')).toEqual('akshansh.verma01@gmail.com');
    });
});