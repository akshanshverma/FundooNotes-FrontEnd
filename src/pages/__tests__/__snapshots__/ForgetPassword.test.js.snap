// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`forgetPassword should render correctly in "debug" mode 1`] = `
<div
  className="maindiv"
>
  <WithStyles(Card)
    id="card"
  >
    <WithStyles(Typography)
      color="primary"
      id="loginT"
    >
      Forget Password
    </WithStyles(Typography)>
    <div
      className="hold"
    >
      <div
        className="inputRsP"
      >
        <Input
          id="emailForPasswordReset"
          label="email"
          name="email"
          onChange={[Function]}
          placeholder="enter email"
          type="text"
        />
        <div
          className="msg"
        />
      </div>
      <div
        className="button"
      >
        <WithStyles(Button)
          color="primary"
          onClick={[Function]}
          type="Submit"
          variant="contained"
        >
          submit
        </WithStyles(Button)>
      </div>
      <div
        className="resetLinkMsg"
      />
      <div
        id="spanDiv"
      >
        <span
          className="goLogin"
        >
          <a
            href="/login"
          >
            Login?
          </a>
        </span>
      </div>
    </div>
  </WithStyles(Card)>
</div>
`;
